### DISCUSSION TITLE
Modding: Beam Weapon Question

### DISCUSSION THREAD

**Discussion by VaelophisNyx** [endless-sky/endless-sky#10582] [Q&A] [answered]
I recall a few years ago when I used to mod ES, that beam weapons needed their sprite length and range to match. Is that still true? (or was it ever true?)

**Comment by TheGiraffe3** [comment, marked as answer]
That is still true.

**Reply by VaelophisNyx** [reply]
Weirdly I was experimenting with it earlier and found that if I used the exact length of an effect, it was 2x the distance it should be from the gun port/turret firing it
Halved the number and its perfect?

**Reply by VaelophisNyx** [reply]
oh this could actually be an issue that I'm running High-DPI and the sprite wasn't made for that
